boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio1 14 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 1";
+		};
+		button1: button_1 {
+			gpios = <&gpio1 13 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 2";
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		red_led: led_1 {
+			gpios = <&gpio1 8 GPIO_ACTIVE_HIGH>;
+			label = "RGB red LED";
+		};
+		green_led: led_2 {
+			gpios = <&gpio1 6 GPIO_ACTIVE_HIGH>;
+			label = "RGB green LED";
+		};
+		blue_led: led_3 {
+			gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
+			label = "RGB blue LED";
+		};
+	};
+
+	pwmdevs {
+		compatible = "pwm-leds";
+		red_led_pwm: led_pwm_0 {
+			pwms = <&pwm0 40>;
+			label = "Red PWM LED";
+		};
+
+		green_led_pwm: led_pwm_1 {
+			pwms = <&pwm0 38>;
+			label = "Green PWM LED";
+		};
+
+		blue_led_pwm: led_pwm_2 {
+			pwms = <&pwm0 39>;
+			label = "Blue PWM LED";
+		};
+
+		buzzer_pwm0: buzzer_pwm_0 {
```

Comment:
I somehow doubt that buzzer is compatible with `pwm-leds`...

Reply:
Im removing it as i cant find suitable compatible.